share/packages/mbgft.xml

```diff
@@ -1,7 +1,7 @@
 <gwbse>
         <dftbasis>ubecppol</dftbasis>
         <gwbasis>aux-ubecppol</gwbasis>
-        <ranges></ranges> <!-- default: all levels in RPA, 1:2*HOMO in QP and BSE; other options: factor,explicit -->
+        <ranges>default</ranges> <!-- default: all levels in RPA, 1:2*HOMO in QP and BSE; other options: factor,explicit -->
```

Comment:
Are the defaults still this way? Should we change them?

Reply:
if you want, the are still [0,homo][homo,2xhomo]